src/utils/glob.js

```diff
@@ -0,0 +1,9 @@
+const isGlob = require('is-glob');
+
+exports.isGlob = function(filename) {
+  return isGlob(filename, {
+    strict: true
```

Comment:
looks like strict is the default: https://github.com/micromatch/is-glob/blob/master/index.js#L24